packages/indexer-common/src/indexer-management/monitor.ts

```diff
@@ -1141,15 +1140,15 @@ Please submit an issue at https://github.com/graphprotocol/block-oracle/issues/n
             ) {
```

Comment:
This is missing the variable declaration for `$lastId`. 
`graph-node` may let it pass, but I think that The Guild is/was working on making graph-node stricter wrt. the GraphQL spec, which could cause issues in the future.